packages/astro/src/client/sdk.ts

```diff
@@ -0,0 +1,42 @@
+import type { BrowserOptions } from '@sentry/browser';
+import { BrowserTracing, init as initBrowserSdk } from '@sentry/browser';
+import { configureScope, hasTracingEnabled } from '@sentry/core';
+import { addOrUpdateIntegration } from '@sentry/utils';
+
+import { applySdkMetadata } from '../common/metadata';
+
+// Treeshakable guard to remove all code related to tracing
+declare const __SENTRY_TRACING__: boolean;
+
+/**
+ * Initialize the client side of the Sentry Astro SDK.
+ *
+ * @param options Configuration options for the SDK.
+ */
+export function init(options: BrowserOptions): void {
+  applySdkMetadata(options, ['astro', 'browser']);
+
+  addClientIntegrations(options);
+
+  initBrowserSdk(options);
+
+  configureScope(scope => {
+    scope.setTag('runtime', 'browser');
+  });
+}
+
+function addClientIntegrations(options: BrowserOptions): void {
+  let integrations = options.integrations || [];
+
+  // This evaluates to true unless __SENTRY_TRACING__ is text-replaced with "false",
+  // in which case everything inside will get treeshaken away
+  if (typeof __SENTRY_TRACING__ === 'undefined' || __SENTRY_TRACING__) {
```

Comment:
could this not also be `if (__SENTRY_TRACING === false)`?

Reply:
Hmm perhaps I'm missing something but do you mean `if (__SENTRY_TRACING !== false)`